pkgs/shells/fish/plugins/puffer.nix

```
@@ -0,0 +1,20 @@
+{ lib, buildFishPlugin, fetchFromGitHub }:
+
+buildFishPlugin rec {
+  pname = "puffer";
+  version = "unstable-2021-05-21";
+
+  src = fetchFromGitHub {
+    owner = "nickeb96";
+    repo = "puffer-fish";
+    rev = "df333fff5130ef8bf153c9bafbf0661534f81d9c";
```

nit (upstream has updated)
```suggestion
  version = "unstable-2022-10-07";

  src = fetchFromGitHub {
    owner = "nickeb96";
    repo = "puffer-fish";
    rev = "fd0a9c95da59512beffddb3df95e64221f894631";
```